be/src/http/action/compaction_action.h

```diff
@@ -75,8 +70,6 @@ class CompactionAction : public HttpHandler {
     static std::mutex _compaction_running_mutex;
     /// whether there is manual compaction running
     static bool _is_compaction_running;
-    /// memory tracker
-    std::shared_ptr<MemTracker> _compaction_mem_tracker;
```

Comment:
why remove _compaction_mem_tracker?

Reply:
Because `_compaction_mem_tracker` is not used,
Compaction memory is tracked by `StorageEngine::instance()->compaction_mem_tracker()`.